src/migrations/migrations/migratePersistedQueriesToMMKV.ts

```diff
@@ -0,0 +1,14 @@
+import { Migration, MigrationName } from '@/migrations/types';
+import { queryClient } from '@/react-query';
+import { queryStorage } from '@/storage/legacy';
+
+export function migratePersistedQueriesToMMKV(): Migration {
+  return {
+    name: MigrationName.migratePersistedQueriesToMMKV,
+    async defer() {
+      // query storage will retreive the whole query state from asyncstorage and replace it in MMKV
+      await queryStorage.get(['rainbow.react-query']);
```

Comment:
this no work, afaik this is what we tested together but no dice 

for testing this, checkout develop, go to swap output currency select, fav a bunch of assets, and then checkout your branch again. you can add `debug: true`to the migration and it'll get run every start up 

also unsure if we should defer this if we are going to migrate over all of the storage at once, would be better to just grab favs and let the rest get overrriden naturally i think

Reply:
you right, fixed it